torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -169,10 +169,17 @@ struct GraphFuser {
   Block* block_;
   std::unique_ptr<AliasDb> aliasDb_;
   std::shared_ptr<Graph> graph_;
+  using FusionCallback = std::function<bool(Node*)>;
```

Comment:
nit: can we move this to not appear inside a sequence of member decalarations? It is quite confusing at the moment